packages/crayons-extended/custom-objects/src/components/form-builder/form-builder.tsx

```diff
@@ -223,6 +227,13 @@ export class FormBuilder {
       'RELATIONSHIP',
       'MULTI_SELECT',
     ];
+
+    if (!this.showLookupField) {
+      this.supportedFieldTypes.splice(
```

Comment:
supportedField types should not be altered. This needs to be controlled in fieldOrder key of form-mapper